src/Access/Dialog.vala

```diff
@@ -75,27 +73,28 @@ public class Access.Dialog : Granite.MessageDialog {
 
         custom_bin.orientation = Gtk.Orientation.VERTICAL;
         custom_bin.spacing = 6;
+    }
+
+    public override void show () {
```

Comment:
Can you explain why it's better to override `show` than to connect to `realize`? Is there an issue that this fixes?

Reply:
i don't think this should change the current behaviour, this is more a organization/simplification change (separate widget code from windowing code, reduce the number of functions used).